tests/html/last_line/jsfmt.spec.js

```diff
@@ -0,0 +1,2 @@
+run_spec(__dirname, ["html"], { jsxBracketSameLine: true });
+run_spec(__dirname, ["html"], { jsxBracketSameLine: false });
```

Comment:
It should be `bracketSameLine` or other name, not `jsxBracketSameLine`, because `jsx` is not `html`, I would advise to do:

First PR.

- Added the new option
- if developer doesn't set `jsxBracketSameLine` we will try value of `bracketSameLine`, if still nothing, use default value

Second PR.

- Improve this for HTML + tests

Third PR.

- Improve this for VUE and etc

Reply:
Thanks, I addressed the naming issue in my post above. The eventual goal is that there is only one option, right? with `jsxBracketSameLine` being either a synonym of `angleBracketSameLine` (or whatever it is called) or deprecated.

Reply:
/cc @prettier/core

Reply:
We  need rename`jsxBracketSameLine` first. Then apply to  html .

Reply:
Let me look into a different pull request that starts with a rename.

Reply:
Would a pull request for a rename include renaming in all the tests and docs? 
also, @alexander-akait , you wrote:
> if developer doesn't set jsxBracketSameLine we will try value of bracketSameLine, if still nothing, use default value
I think we could simplify that to "If a developer sets either of the settings to true, we print the bracket on the same line." The default is false, and we don't want to inadvertently create a new option here by treating jsxBracketSameLine differently from angleBracketSameLine. We are renaming an old option, not adding a new one.